driver-core/src/test/java/com/datastax/driver/core/ClusterInitTest.java

```diff
@@ -116,7 +117,9 @@ public void should_handle_failing_or_missing_contact_points() throws UnknownHost
             // - 0 or 1 for the missing host. We can't know for sure because contact points are randomized. If it's tried
             //   before the live host there will be a connection attempt, otherwise it will be removed directly because
             //   it's not in the live host's system.peers.
+            //noinspection ResultOfMethodCallIgnored
             verify(socketOptions, atLeast(6)).getKeepAlive();
+            //noinspection ResultOfMethodCallIgnored
             verify(socketOptions, atMost(7)).getKeepAlive();
```

Comment:
I know but there is no standard equivalent for this warning, it's specific to IntelliJ. In hindsight, I think the right choice is to remove these comments altogether.